packages/components/pie-checkbox/test/component/pie-checkbox.spec.ts

```diff
@@ -330,45 +329,6 @@ test.describe('PieCheckbox - Component tests', () => {
                 expect(assistiveText).not.toBeVisible();
             });
 
-            test('should apply the "default" variant attribute if no status is provided', async ({ mount, page }) => {
```

Comment:
These are covered by visual tests

Reply:
I'm unsure what benefit we think this test provides given we visually test it looks correct when no status is provided to the checkbox. Is there a reason we want to test this attribute @kevinrodrigues?

Reply:
Im ok with it being deleted. I do see the benefit of having this test tho.